test/integration/image_test.go

```diff
@@ -0,0 +1,132 @@
+//go:build integration
+
+/*
+Copyright 2022 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+func TestImageBuild(t *testing.T) {
+	type validateFunc func(context.Context, *testing.T, string)
+	profile := UniqueProfileName("image")
+	ctx, cancel := context.WithTimeout(context.Background(), Minutes(15))
+	defer Cleanup(t, profile, cancel)
+
+	// Serial tests
+	t.Run("serial", func(t *testing.T) {
+		tests := []struct {
+			name      string
+			validator validateFunc
+		}{
+			{"NormalBuild", validateNormalImageBuild},
+			{"BuildWithBuildArg", validateImageBuildWithBuildArg},
+			{"BuildWithDockerIgnore", validateImageBuildWithDockerIgnore},
+			{"BuildWithSpecifiedDockefile", validateNormalImageBuildWithSpecifiedDockefile},
+		}
+		for _, tc := range tests {
+			if ctx.Err() == context.DeadlineExceeded {
+				t.Fatalf("Unable to run more tests (deadline exceeded)")
+			}
+			if t.Failed() {
+				t.Fatalf("Previous test failed, not running dependent tests")
+			}
+
+			t.Run(tc.name, func(t *testing.T) {
+				tc.validator(ctx, t, profile)
+				if t.Failed() && *postMortemLogs {
+					PostMortemLogs(t, profile)
+				}
+			})
+		}
+	})
+}
+
+// normal test case for minikube image build, with -t parameter
+func validateNormalImageBuild(ctx context.Context, t *testing.T, profile string) {
+	defer PostMortemLogs(t, profile)
+	args := append([]string{"image", "build", "-t", "aaa:latest", "./testdata/image-build/test-normal", "-p", profile}, StartArgs()...)
```

Comment:
Start args are only when starting minikube.
```suggestion
	args := []string{"image", "build", "-t", "aaa:latest", "./testdata/image-build/test-normal", "-p", profile}
```